toast.css

```diff
@@ -0,0 +1,59 @@
+.toast {
+  background-color: azure;
+  min-width: 150px;
+  height: 30px;
+  font-family: "Roboto";
+  font-size: 20px;

```

Comment:
It is recommended to use rem units instead of px for values above 10px-16px. 
Use color variables for better flexibility.